src/jit/titypes.h

```diff
@@ -6,6 +6,7 @@ DEF_TI(TI_ERROR, "<ERROR>")
 DEF_TI(TI_REF, "Ref")
 DEF_TI(TI_STRUCT, "Struct")
 DEF_TI(TI_METHOD, "Method")
+DEF_TI(TI_TOKEN, "Token")
```

Comment:
I would add a flag to `typeInfo::m_bits` rather than adding a new `TI_type`:
a) the `TI_TOKEN` case is logically a specialization of `TI_METHOD`, and so doesn't require a new type
b) adding a new `TI_type` impacts more code than just that which uses `typeInfo` values